dbetka/harcmap
# Issue/675/create script to set default es lint and webpack configuration in intellij ide

**Plugin docs:**
https://github.com/dbetka/wdk/blob/main/docs/init-intellij-settings.md

**Steps to test:**
1. Init new harcmap repository.
```bash
git clone git@github.com:dbetka/harcmap.git
// or
git clone https://github.com/dbetka/harcmap.git
```
2. Go to branch `issue/675/Create-script-to-set-default-ESLint-and-Webpack-configuration-in-Intellij-IDE`
3. Make `npm i` in root project directory.
4. Make `npm run init-intellij-settings`.
5. Restart IDE.
6. Check in settings if eslint and webpack have correct settings.
![image](https://user-images.githubusercontent.com/6503362/188183425-40337448-a159-4e71-aacc-601f0ee52bd7.png)
![image](https://user-images.githubusercontent.com/6503362/188183415-c337676d-cd76-4600-aab5-12b8c01930e1.png)
7. Test if eslint rules works properly.
8. Check if you can jump to source through import paths.
> File: client/src/index.js

  ![image](https://user-images.githubusercontent.com/6503362/188183847-98fc1699-684d-4cd8-a798-718b0b6b58a7.png)
